standalone/runtime/src/lib.rs

```diff
@@ -1525,6 +1525,36 @@ impl pallet_oracle::Config for Runtime {
 impl puppets::parachain_info::Config for Runtime {}
 impl puppets::parachain_system::Config for Runtime {}
 
+type CreditForServiceFee = frame_support::traits::tokens::fungible::Credit<AccountId, Balances>;
+
+pub struct DealWithServiceFee;
+impl OnUnbalanced<CreditForServiceFee> for DealWithServiceFee {
+	fn on_nonzero_unbalanced(amount: CreditForServiceFee) {
+		drop(amount);
+	}
+}
+
+parameter_types! {
+    pub EIP712Name: Vec<u8> = b"Substrate".to_vec();
+    pub EIP712Version: Vec<u8> = b"1".to_vec();
+    pub EIP712ChainID: pallet_evm_account_mapping::EIP712ChainID = sp_core::U256::from(0);
+    pub EIP712VerifyingContractAddress: pallet_evm_account_mapping::EIP712VerifyingContractAddress = sp_core::H160::from([0u8; 20]);
+}
+
+impl pallet_evm_account_mapping::Config for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type RuntimeCall = RuntimeCall;
+    type Currency = Balances;
+    type ServiceFee = ConstU128<1000>;
```

Comment:
0.000000001 PHA?

Reply:
Changed to `100000000` should be 0.0001 PHA?

Reply:
I think it can be 0.001 or 0.01 PHA. It's not good to make it too small. Btw, why not send the service fee to the treasury?

Reply:
This piece of code is copied from node-template, which doesn't have treasury, I can use treasury here

Reply:
Updated